app/views/static_pages/landing.html.haml

```diff
@@ -1,88 +1,73 @@
 .landing
   .row.landing-banner
+    .col-md-12.landing-banner-background
     .col-md-12
-      = image_tag 'icon.png', alt: 'RubyBench Logo', class: 'landing-logo'
+      = image_tag('icon.png', alt: 'RubyBench Logo', class: 'landing-logo')
```

Comment:
haha I actually removed the brackets in another commit. Just a style that I follow xD